### DISCUSSION TITLE
Allow us to freely move status bars / put them on the right side / change height

### DISCUSSION THREAD

**Discussion by clownshoe123** [runelite/runelite#15988] [Ideas]
Since we can move our inventory now it would be handy if the status bars which are currently fixed to the left side of our inventory had more customization. I say this because generally if you're moving your inventory it's probably so you can bank/click something easier and the status bars overlap the deposit all/withdraw item button ect which would be avoided if you could simply move them. Thanks :)

This is mainly intended for resizable modern layout btw, not sure how the other layouts would function though could maybe just leave them alone if possible.

Also not of high importance but might as well mention just incase you feel like it, changing the height could come in handy for some situations

**Comment by Quopsmall** [comment]
Yes, I have been hoping for the same thing. Would like to be able to move the status bars with the ALT key or in the plug in configuration have an option to toggle either 'left' or 'right' of the inventory

![Status Bar](https://github.com/runelite/runelite/assets/141461595/e32861b1-8576-403d-aacc-1b03c4ef8473)

![Status Bar 2](https://github.com/runelite/runelite/assets/141461595/56db291e-f3ff-4720-9e20-b2d7406cf03a)

**Comment by ZebGasm** [comment]
i also would like if you could have the choice to make them horizontal and resize them
